src/core/expr/fexpr_cut.cc

```diff
@@ -54,6 +58,8 @@ class FExpr_Cut : public FExpr_Func {
       if (!py_nbins_.is_none()) {
         out += ", nbins=";
         out += py_nbins_.repr().to_string();
+        out += ", bins=";
+        out += py_bins_.repr().to_string();
```

Comment:
we  probably don't want to repr parameters that are at their default values (such as `None`). Especially since having `bins` and `nbins` at the same time is not allowed

Reply:
It seems that the logic here is wrong even for `right_closed`, I will fix it.